filter_plugins/oo_filters.py

```diff
@@ -18,8 +18,8 @@
 from distutils.util import strtobool
 from distutils.version import LooseVersion
 from operator import itemgetter
+from ansible.module_utils.six.moves.urllib.parse import urlparse
```

Comment:
That's required for it to work. It's actually part of another PR open right now. I'll remove it once the other PR merges.